macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -666,6 +667,36 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
     }
 
     private func start(_ tunnelManager: NETunnelProviderManager) async throws {
+
+        let options = try await prepateStartupOptions()
+
+        if Self.simulationOptions.isEnabled(.controllerFailure) {
+            Self.simulationOptions.setEnabled(false, option: .controllerFailure)
+            throw StartError.simulateControllerFailureError
+        }
+
+        do {
+            Logger.networkProtection.log("🚀 Starting NetworkProtectionTunnelController, options: \(options, privacy: .public)")
+            try tunnelManager.connection.startVPNTunnel(options: options)
+
+            if #available(macOS 12, *) {
+                try await waitForStartSuccess(tunnelManager)
+            }
```

Comment:
This is also new and will ensure we capture additional failures (like silent VPN startup failures) in pixels.